docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Clustering/KMeansWithOptions.tt

```diff
@@ -0,0 +1,33 @@
+﻿<#@ include file="Clustering.ttinclude"#>
+<#+
+string ClassName = "KMeansWithOptions";
+string Trainer = "KMeans";
+string TrainerOptions = @"KMeansTrainer.Options
+            {
+                NumberOfClusters = 2,
+                MaximumNumberOfIterations = 100,
```

Comment:
>MaximumNumberOfIterations = 100 [](start = 16, length = 31)

I don't think we should suggest tuning this. Mostly we want this option to *not* be hit.